CMake/External_Darknet.cmake

```diff
@@ -0,0 +1,89 @@
+set(Darknet_DEPENDS)
+
+if(fletch_ENABLE_OpenCV)
+  option(fletch_ENABLE_Darknet_OpenCV "Build Darknet with OpenCV support" TRUE )
+  mark_as_advanced(fletch_ENABLE_Darknet_OpenCV)
+else()
+  unset(fletch_ENABLE_Darknet_OpenCV CACHE)
+endif()
+if(fletch_ENABLE_Darknet_OpenCV)
+  set(DARKNET_OPENCV_ARGS -DUSE_OPENCV:BOOL=ON)
+  add_package_dependency(
+    PACKAGE Darknet
+    PACKAGE_DEPENDENCY OpenCV
+    PACKAGE_DEPENDENCY_ALIAS OpenCV
+    )
+  set(Darknet_DEPENDS ${Darknet_DEPENDS} OpenCV)
```

Comment:
I believe the add_package_dependency macro adds the package to Darknet_DEPENDS making this line redundant. You can message to prove that.